server/Dockerfile

```diff
@@ -108,12 +108,14 @@ RUN ./configure-apt.sh && \
     apt-get remove -yqq jq wget ca-certificates && \
     apt-get autoremove -yqq && \
     apt-get clean && \
-    dpkg -r --force-depends libjpeg62-turbo && \
     rm -rf \
         configure-apt.sh \
         install-ffmpeg.sh \
         *.deb \
-        /var/lib/apt/lists
+        /var/lib/apt/lists \
+        /usr/lib/$(arch)-linux-gnu/libjpeg.so* \
```

Comment:
Why do you want to remove libjpeg?
Sharp does not use by default the decoder from `/usr/local/lib`?

```bash
root@03fa65ee8b06:/# ldd /usr/local/bin/vips | grep jpeg
        libjpeg.so.8 => /usr/local/lib/libjpeg.so.8 (0x000014f26b39f000)
```

Reply:
It makes the image smaller and avoids any ambiguity.